### PR TITLE
ACPI: add rt5682 acpi for ICL RVP

### PR DESCRIPTION
**Pull request by bardliao**
add rt5682 acpi for ICL RVP

Signed-off-by: Bard liao <yung-chuan.liao@linux.intel.com>

### CODE REVIEW

**Comment by bardliao** [comment]
@plbossart It seems I can't add reviewers. Could you review it? Thanks.

File: ICL/rt5682.asl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                Method (_CRS, 0, Serialized)  // _CRS: Current Resource Settings
+                {
+                    Name (SBUF, ResourceTemplate ()
+                    {
+                        I2cSerialBus (0x001A, ControllerInitiated, 0x00061A80,
+                            AddressingMode7Bit, "\\_SB.PCI0.I2C0",
+                            0x00, ResourceConsumer, ,
+                            )
+                    })
+                    Return (SBUF) /* \_SB_PCI0.I2C1.P512._CRS.SBUF */
```
Comment (plbossart) [inline ICL/rt5682.asl]: copy-paste here?

**Review by plbossart — COMMENTED** [review]
minor copy-paste below

File: ICL/rt5682.asl (hunk 1 above)
Comment (bardliao) [reply ICL/rt5682.asl]: @plbossart Yes, the file is copy-pasted from PCM512X.asl. Thanks for the review. The PR has updated.

**Review by plbossart — APPROVED** [review]

**Pull request merged**